projects/ngrx.io/content/guide/store/configuration/runtime-checks.md

```diff
@@ -165,3 +165,4 @@ function logTodo (todo: Todo) {
   console.log(todo);
 }
 ```
+Please note, you may not need to set `strictActionSerializability` to `true` unless you are storing/replaying actions using external resources, for example `localStorage`.
```

Comment:
Let's wrap this in an alert div.

```suggestion

<div class="alert is-important">

Please note, you may not need to set `strictActionSerializability` to `true` unless you are storing/replaying actions using external resources, for example `localStorage`.

</div>
```

Reply:
just came back vacation, will do shortly.